src/Wizard/Wizard.test.js

```diff
@@ -0,0 +1,252 @@
+            { activeStepIndex, activeSubStepIndex },
+            onStepClick,
+            onSidebarItemClick,
+            onBackButtonClick,
+            onNextButtonClick
+          ) => {
+            return (
+              <div>
+                <Wizard.Header title="Wizard Title" />
+                <Wizard.Body>
+                  <Wizard.Steps
+                    steps={renderWizardSteps(
+                      mockWizardItems,
+                      activeStepIndex,
+                      activeSubStepIndex,
+                      onStepClick
+                    )}
+                  />
+                  <Wizard.Row>
+                    <Wizard.Sidebar
+                      items={renderSidebarItems(
+                        mockWizardItems,
+                        activeStepIndex,
+                        activeSubStepIndex,
+                        onSidebarItemClick
+                      )}
+                    />
+                    <Wizard.Main>
+                      {renderWizardContents(
+                        mockWizardItems,
+                        activeStepIndex,
+                        activeSubStepIndex
+                      )}
+                    </Wizard.Main>
+                  </Wizard.Row>
+                </Wizard.Body>
+                <Wizard.Footer>
+                  <Button bsStyle="default" className="btn-cancel">
+                    Cancel
+                  </Button>
+                  <Button
+                    bsStyle="default"
+                    disabled={activeStepIndex === 0 && activeSubStepIndex === 0}
+                    onClick={onBackButtonClick}
+                  >
+                    <span className="i fa fa-angle-left" />Back
+                  </Button>
+                  {(activeStepIndex === 0 || activeStepIndex === 1) && (
+                    <Button bsStyle="primary" onClick={onNextButtonClick}>
+                      Next<span className="i fa fa-angle-right" />
+                    </Button>
+                  )}
+                  {activeStepIndex === 2 &&
+                    activeSubStepIndex === 0 && (
+                      <Button bsStyle="primary" onClick={onNextButtonClick}>
+                        Deploy<span className="i fa fa-angle-right" />
+                      </Button>
+                    )}
+                  {activeStepIndex === 2 &&
+                    activeSubStepIndex === 1 && (
+                      <Button bsStyle="primary">
+                        Done<span className="i fa fa-angle-right" />
+                      </Button>
+                    )}
+                </Wizard.Footer>
+              </div>
+            )
+          }}
+        />
+      </Col>
+    </Row>
+  )
+
+  const tree = component.toJSON()
+  expect(tree).toMatchSnapshot()
+})
+
+/**
+ * react-overlays / Jest opened issue
+ * Open issue: https://github.com/react-bootstrap/react-overlays/issues/225
+ */
+// test('Wizard modal renders properly', () => {
```

Comment:
csn use xit instead to comment out the code. (not sure whats better tbh)

Reply:
It looks like I can remove this test and convert to Enzyme now